pkgs/kde/misc/marknote/default.nix

```diff
@@ -0,0 +1,34 @@
+{
+  lib,
+  mkKdeDerivation,
+  fetchurl,
+  extra-cmake-modules,
+  kcrash,
+  qtdeclarative,
+  qtsvg,
+  qtwayland,
+  qqc2-desktop-style
+}:
+mkKdeDerivation rec {
+  pname = "marknote";
+  version = "1.2.1";
+
+  src = fetchurl {
+    url = "mirror://kde/stable/marknote/marknote-${version}.tar.xz";
+    hash = "sha256-HzImkm8l8Rqiuyq2QezfdqJ1hxIdLZhiIGVM9xzpyaA=";
+  };
+
+  extraNativeBuildInputs = [
+    extra-cmake-modules
```

Comment:
Should be already there via mkKdeDerivation.

Reply:
true. fixed. thx.